plat/drivers/virtio/virtio_net.c

```diff
@@ -898,12 +898,12 @@ static int virtio_netdev_feature_negotiate(struct uk_netdev *n)
 	/**
 	 * MTU information (needed)
 	 */
-	if (!VIRTIO_FEATURE_HAS(host_features, VIRTIO_NET_F_STATUS)) {
+	if (!VIRTIO_FEATURE_HAS(host_features, VIRTIO_NET_F_MTU)) {
 		uk_pr_err("%p: Host system does not offer MTU feature\n", n);
 		rc = -EINVAL;
 		goto err_negotiate_feature;
 	}
-	VIRTIO_FEATURE_SET(drv_features, VIRTIO_NET_F_STATUS);
+	VIRTIO_FEATURE_SET(drv_features, VIRTIO_NET_F_MTU);
```

Comment:
Hi @dinhngtu, I think you should also add the check for `VIRTIO_NET_F_STATUS` . that way, you can check the link status during initialization.
_reference virtio v1.1 specs, section 5.1.4.2_

Reply:
Hey, @Sairajkodilkar. Yeah, I agree, I think that should make sense.

Reply:
`VIRTIO_NET_F_MTU` seems necessary since `uk_netdev_ops` has the `mtu_get` operation (but the wiring for MTU reporting doesn't seem to exist ATM). I'll change `virtio_net_mtu_get` to report the actual device MTU.

Reply:
During feature negotiation, you can see whether the MTU is available. If it is available, then read the MTU value; otherwise, assign the default MTU value